be/tool/beck.c

```diff
@@ -213,6 +215,8 @@ struct action_ops {
 };
 
 enum { CACHE_SIZE = 1000000 };
+enum { NV_OFFSET_SAVE_DELTA_IN_BYTES = 0x40000000 }; /* 1G */
+enum { NV_OFFSET_SAVE_ACT_DELTA = 1000 };
```

Comment:
Is there any Performance Impact of this?

Reply:
yes, as this is additional processing there will  performance impact, we can try to reduce by increasing delta constants but the penalty will be we may restart processing too much back.